client/my-sites/sidebar-unified/index.jsx

```diff
@@ -71,17 +96,27 @@ export const MySitesSidebarUnified = ( { path } ) => {
 								link={ item.url }
 								selected={ isSelected }
 								sidebarCollapsed={ sidebarIsCollapsed }
+								continueInCalypso={ continueInCalypso }
 								{ ...item }
 							/>
 						);
 					}
 
 					return (
-						<MySitesSidebarUnifiedItem key={ item.slug } selected={ isSelected } { ...item } />
+						<MySitesSidebarUnifiedItem
+							key={ item.slug }
+							selected={ isSelected }
+							continueInCalypso={ continueInCalypso }
+							{ ...item }
+						/>
 					);
 				} ) }
 				<CollapseSidebar key="collapse" title="Collapse menu" icon="dashicons-admin-collapse" />
 			</Sidebar>
+			<ExternalLinkDialog
```

Comment:
Shall we `AsyncLoad` this dialog? I can't imagine the vast amount of users will need this immediately loaded in?

Reply:
Good point, 22ee526!